AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -22,53 +25,76 @@ extern BOOL CGPointIsNull(CGPoint point)
   return isnan(point.x) && isnan(point.y);
 }
 
+@interface ASLayout ()
+
+/**
+ * A boolean describing if the current layout has been flattened.
+ */
+@property (nonatomic, getter=isFlattened) BOOL flattened;
```

Comment:
That's a great change as some dev's already asked for that property and actually wanted to use it for some reason